### ISSUE TITLE
Specify certificate based authentication with cert thumbprint

### ISSUE THREAD

**Issue by BrianTJackett** [microsoftgraph/msgraph-sdk-powershell#34]
Suggestion to support specifying the thumbprint of certificate to be used for certificate based authentication.  This would be in addition to (or replacing) the option for certificate name.  If I'm not mistaken the certificate name could be the same across many different certificates (like a display name) vs. the thumbprint which uniquely identifies the certificate (like an identifier).

Ex. Connect-Graph -ClientId "[clientId]" -TenantId "[tenantId]" -CertificateThumbprint "[certThumbprint]"

**Comment by peombwa** [comment]
@darrelmiller, as part of this work, should we expose an optional `-CertificatePath` parameter to `Connect-Graph` cmdlet? This will be used for scenarios where a customer would like to use a certificate in a different path other than `cert:\CurrentUser\My`. The default path will still be `cert:\CurrentUser\My` when `-CertificatePath` is not specified.

**Comment by darrelmiller** [comment]
@peombwa  I don't have any objections to that plan, but I do wonder if it is necessary.  Have you seen examples of other tools doing something similar?

**Comment by peombwa** [comment]
@darrelmiller, this was in relation to this issue https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/52 where the customer typically stores their certs in `cert:\LocalMachine\My`.

Nearly all tools ([Azure SDK](https://docs.microsoft.com/en-us/powershell/module/az.accounts/connect-azaccount?view=azps-3.1.0), [AzureAD docs](https://github.com/Azure-Samples/active-directory-dotnetcore-daemon-v2/blob/5199032b352a912e7cc0fce143f81664ba1a8c26/AppCreationScripts-withCert/Configure.ps1#L164)) make the assumption that certs will be stored in `cert:\CurrentUser\My`. For consistency purposes with other tools, we can maintain the implementation as it is and only use `cert:\CurrentUser\My`.

**Comment by peombwa** [comment]
This has been resolved in version 0.1.4.
``` ps
Install-Module Microsoft.Graph.Beta -RequiredVersion 0.1.4 -Repository GraphPowerShell -Force -AllowClobber
```
![image](https://user-images.githubusercontent.com/7061532/70751401-2e69de00-1ce5-11ea-86e8-5594c59a16ea.png)

**Comment by BrianTJackett** [comment]
Confirmed this is working as requested in 0.1.4.  Thanks so much for getting this released.
